test/test_scanner.py

```diff
@@ -666,6 +681,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "gnutls-cli",
                         "2.3.11",
                     ),
+                    (
+                        "https://ftp.gnu.org/gnu/wget/",
+                        "wget-1.11.2.tar.bz2",
```

Comment:
Does this contain a compiled version of the code, or only the source code?  cve-**bin**-tool only works on binaries, so you'll need a compiled binary to test against.